### PR TITLE
Use distinct droppable trash ID.

### PR DESCRIPTION
**Pull request by WoltersoderWolltersnicht**

### CODE REVIEW

**Comment by samcf** [comment]
Thanks for the PR! I will take a look at this shortly 👍

File: src/main/ogres/app/component/panel_tokens.cljs [hunk 1]
```diff
@@ -179,10 +183,13 @@
...
-                    "scope-pub" (dispatch :token-images/change-scope drag :public)
-                    "scope-prv" (dispatch :token-images/change-scope drag :private)
-                    "trash" (dispatch :token-images/remove drag nail))
+                  (do
+                    (if (= drop "scope-pub")
+                      (dispatch :token-images/change-scope drag :public))
+                    (if (= drop "scope-prv")
+                      (dispatch :token-images/change-scope drag :private))
+                    (if (str/includes? drop "trash")
+                      (dispatch :token-images/remove drag nail)))
```
Comment (samcf) [inline src/main/ogres/app/component/panel_tokens.cljs:192]:
```suggestion
                  (cond
                    (= drop "scope-pub")
                    (dispatch :token-images/change-scope drag :public)
                    (= drop "scope-prv")
                    (dispatch :token-images/change-scope drag :private)
                    (str/includes? drop "trash")
                    (dispatch :token-images/remove drag nail))
```

File: src/main/ogres/app/component/panel_tokens.cljs [hunk 2]
```diff
@@ -61,7 +62,10 @@
      (.querySelector js/document "#root"))))
 
 (defui ^:private gallery [props]
-  (let [option (use-droppable #js {"id" "trash"})]
+  (let [name (:name props)
+        dynamic-id (str name "-" "trash")
+        option (use-droppable #js {"id" dynamic-id})
+        ]
```
Comment (samcf) [inline src/main/ogres/app/component/panel_tokens.cljs:68]:
```suggestion
  (let [name (:name props)
        dynamic-id (str name "-" "trash")
        option (use-droppable #js {"id" dynamic-id})]
```

**Review by samcf — COMMENTED** [review]
This looks really good! This solves an annoying issue thats been around for a while. Just made two minor suggestions for code quality, I'll merge if you think that looks good!

**Comment by WoltersoderWolltersnicht** [comment]
It's my first time coding in clojure, so all your code suggestions are great. 😃👍🏻 
Thanks for reviewing

**Pull request merged**